llvm/unittests/Transforms/Scalar/LoopPassManagerTest.cpp

```diff
@@ -328,6 +329,17 @@ class LoopPassManagerTest : public ::testing::Test {
   }
 };
 
+template <size_t Tag, typename MemberPtrT> struct PrivateVisitor {
+  inline static MemberPtrT Ptr;
+};
+template <size_t Tag, auto MemberPtrV> struct PrivateVisitorHelper {
+  struct Assigner {
+    Assigner() { PrivateVisitor<Tag, decltype(MemberPtrV)>::Ptr = MemberPtrV; }
+  };
+  inline static Assigner A;
+};
+template struct PrivateVisitorHelper<0, &LoopPassManager::IsLoopNestPass>;
```

Comment:
I couldn't find a better way to do that, `printPipeline` doesn't show separators between different pass managers.